### DISCUSSION TITLE
Patches for Facebook

### DISCUSSION THREAD

**Discussion by Azharlashari** [ReVanced/revanced-discussions#723] [Question] [answered]
Any plan to provide patches for facebook just like new versions of facebook doesn't support audio live but old versions supports audio live but old versions are so buggy and so many patches are usefull for facebook just like aero instagram

**Comment by reisxd** [comment, marked as answer]
React Native apps can't be patched.

**Reply by Azharlashari** [reply]
What about Instagram?

**Reply by reisxd** [reply]
That's also a React Native application.

**Reply by Azharlashari** [reply]
But instagram has so many mods like aero instagram so many awesome features do you don't know?

**Reply by ghost** [reply]
We do know but ReVanced patcher cannot patch React Native apps at the moment.

**Reply by Azharlashari** [reply]
Any plane for feature?

**Reply by ghost** [reply]
No plans currently.
